tests/test_interpolation_cubic.py

```diff
@@ -0,0 +1,269 @@
+import numpy as np
```

Comment:
IMHO this file should be squashed with `test_interpolation`, and as much code and existing tests as possible should be reused, through `pytest.parametrize` for example